### ISSUE TITLE
zwave part missing

### ISSUE THREAD

**Issue by goyaweb** [tmsns/homey-json-schema#1]
https://developers.athom.com/library/zwave/

**Comment by tmsns** [comment]
Yup, should add them. I'll try to get on it during the holidays. Also, I think there is a new version I need to support as well. :-)
